src/views/TradingCompetition/components/CompetitionHeadingText.tsx

```diff
@@ -0,0 +1,69 @@
+import { Heading } from '@pancakeswap-libs/uikit'
+import styled from 'styled-components'
+
+const H1SizeStyles = (theme) => `
+font-size: 48px;
+line-height: 110%;
+white-space: nowrap;
+
+${theme.mediaQueries.sm} {
+  font-size: 64px;
+}
+`
+
+const H2SizeStyles = (theme) => `
+font-size: 32px;
+line-height: 110%;
+white-space: nowrap;
+
+${theme.mediaQueries.sm} {
+  font-size: 40px
+}
+`
+
+const sharedStyles = (props) => `
+color: ${props.textColor ? props.textColor : '#ffffff'};
+background:  ${props.background ? props.background : '-webkit-linear-gradient(#7645d9 0%, #452a7a 100%)'};
+background-clip: text;
+-webkit-background-clip: text;
+${
+  props.fill
+    ? `-webkit-text-fill-color: transparent;`
+    : `
+-webkit-text-stroke: 4px transparent;
+text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);`
+}
+`
+
+const sharedVisiblyHiddenStyles = `
+visibility: hidden;
```

Comment:
is this prettified ? We should use the same indentation level than in styled components

Reply:
Yea nice - prettier is happy with this junk styling. I've manually set the indents 👍